pkgs/development/python-modules/flash-attn/default.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  einops,
+  ninja,
+  setuptools,
+  torch,
+  transformers,
+  triton,
+  cudaPackages,
+  config,
+  cudaSupport ? config.cudaSupport,
```

Comment:
If the package is cuda-only (no rocm and no cpu for example) we might as well ignore these flags (except for `cudaCapabilities`)